src/compiler/compile/render_dom/Block.ts

```diff
@@ -378,6 +410,7 @@ export default class Block {
 			l: ${properties.claim},
 			h: ${properties.hydrate},
 			m: ${properties.mount},
+			b: ${properties.bubble},
```

Comment:
only add `b: ${properties.bubble},` when necessary?

```js
return_value.properties.push(p`b: ${properties.bubble}`);
```

Reply:
With that approach I would need to check that the function exists before I call it. I'm not really sure which is better. Setting to `noop` is the pattern the rest of the properties use. idk